SeleniumSolarTechnology/SpecFlowSeleniumSolarTechnology/Features/Cart.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: Cart
+
+@mytag
+Scenario: Add and remove products from cart
+	    Given Open Solartechnology Shop page
+    And Open Cable And Switching link
+	    When Add product item to cart(1)
```

Comment:
It is not clear what this number means. Add one product? Add first prosuct? Add second product (if we count from 0)?